pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -1351,6 +1352,11 @@ async def parse_advisories(self, result):
 
                 for collection in update.collections:
                     coll_dict = UpdateCollection.createrepo_to_dict(collection)
+                    if coll_dict["name"] is None:
+                        coll_dict["name"] = (
+                            "collection-"
```

Comment:
Added a prefix to the generated data but it is open for suggestion if there is a better name for it.

Reply:
I don't mind this name but it could possibly be made more specific, like `collection-autofill-$(12 digits of a hash)`

Reply:
I can do that seems reasonable